solanches/tests/test_controller.py

```diff
@@ -148,3 +147,90 @@ def test_remove_comercio_sucesso(mock_remove_comercio, mock_get_by_name,  contro
     mock_remove_comercio.return_value = 1
     result = controller.remove_comercio(comercio_nome)
     assert result == 1
+
+
+@pytest.mark.skip(reason="TODO")
+@mock.patch('solanches.models.Comercio.get_by_name')
+@mock.patch('solanches.models.Cardapio.get_by_id')
+def test_cadastra_comercio(mock_get_by_name, mock_get_by_id, controller):
```

Comment:
A ordem dos parâmetros dos mocks está ao contrário. A ordem deve ser de baixo pra cima, ou seja:
```suggestion
def test_cadastra_comercio(mock_get_by_id, mock_get_by_name, controller):
```

Além disso, não faz sentido mockar método algum da classe Cardapio pois ela não é chamada em nenhum momento na função.